ports/espressif/boards/adafruit_qualia_s3_rgb666/mpconfigboard.h

```diff
@@ -32,9 +32,15 @@
 #define DEFAULT_I2C_BUS_SDA (&pin_GPIO8)
 #define DEFAULT_I2C_BUS_SCL (&pin_GPIO18)
 
-#define MICROPY_HW_NEOPIXEL (&pin_GPIO4) // also DBLTAP
+#define DEFAULT_SPI_BUS_SCK (&pin_GPIO5)
+#define DEFAULT_SPI_BUS_MISO (&pin_GPIO6)
+#define DEFAULT_SPI_BUS_MOSI (&pin_GPIO7)
 
-#define DOUBLE_TAP_PIN (&pin_GPIO4) // also NEOPIXEL
+// UART pins attached to the USB-serial converter chip
+#define CIRCUITPY_CONSOLE_UART_TX (&pin_GPIO43)
+#define CIRCUITPY_CONSOLE_UART_RX (&pin_GPIO44)
```

Comment:
What converter chip? I didn't think this board had one.

Reply:
The TX pin is on the header. There's no RX pin, though. I can remove this if it doesn't make sense.

Reply:
I don't usually set CONSOLE for that. `DEBUG=1` will enable IDF output to the pin anyway.

Reply:
Setting the console will block its use from other things. If RX isn't pulled down, it could cause spurious input chars into CP too.

Reply:
removed!